src/Compatibility/Microsoft.DotNet.ApiCompatibility/Rules/EnumValuesMustMatch.cs

```diff
@@ -18,32 +17,36 @@ public class EnumValuesMustMatch : Rule
     {
         public override void Initialize(RuleRunnerContext context) => context.RegisterOnTypeSymbolAction(RunOnTypeSymbol);
 
+        private bool isEnum(ITypeSymbol sym) => sym is not null && sym.TypeKind == TypeKind.Enum;
```

Comment:
According to our capitalization conventions, you want to PascalCase the method name: `IsEnum(...)`